vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,54 +1,97 @@
+You'll there is a `+` sign next to `italian` and `desserts` when viewing in the lookup or Tree View. The plus sign indicates that this note is a stub. A **[[stub|dendron://dendron.dendron-site/dendron.concepts#stubs]]** is a placeholder for a note that hasn't been created. Dendron uses stubs to avoid cluttering your file system with empty notes when creating hierarchies.
 
 ![Tree View](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-tree-view.png)
 
-Following this recipes example, you can try creating other notes that might be relevant. Some examples: `recipes.italian.appetizers`, `recipes.chinese`, `recipes.chinese.appetizers`, `recipes.ingredients.favorites`, `recipes.ingredients.shopping-list`. Hierarchies are flexible, and you can create them in whichever way works best for you. Dendron allows you to easily change the hierarchies later if you change your mind on how to organize your notes.
+Following this recipes example, you can create other related. Some examples:
+
+```
+recipes.italian.appetizers
+recipes.chinese
+recipes.chinese.appetizers
+recipes.ingredients.favorites
+recipes.ingredients.shopping-list
+```
+
+Hierarchies are flexible, and you can create them in whichever way works best for you. Dendron allows you to [[easily change the hierarchies later|dendron://dendron.dendron-site/dendron.topic.refactoring]] if you change your mind on how to organize your notes.
 
 ### Finding Notes
 
 To find notes, we use the same lookup interface that we used to create them.
 
-> 🌱 Open Lookup, and type `vege`. This will find your `recipes.vegetarian` note. Hit `Enter` to open that note.
+> 🌱 **ACTION:** Open Lookup, and type `vege`. This will find your `recipes.vegetarian` note. Hit `Enter` to open that note.
+
+Lookup uses _fuzzy search_, which means you can type out partial queries and still see the results. Searching with `*` wildcards is supported. Also entering multiple keywords delimited by space will lookup matching notes regardless of order of the keywords.
+
+- Example: `vege rec` or `rec vege` will match the `recipes.vegetarian` note.
 
-The lookup uses fuzzy search which means you can type out partial results and still see the results. Searching with * wildcards is supported. Also entering multiple keywords delimited by space will look up matching notes regardless of order of the keywords. Example: `vege rec`/`rec vege` will both match `recipes.vegetarian` note. Further info on looking up notes can be found in [[dendron.topic.lookup.find]]
+Further info on using lookup to find notes can be found in [[dendron://dendron.dendron-site/dendron.topic.lookup.find]].
 
 When combined with hierarchies, this fast lookup system enables you to find your notes very quickly, even if you have thousands of notes in your vaults.
 
-If you want to search for content within the notes, you can use VS Code's built-in search tools with `%KEYBINDING%+Shift+F`.
+![[dendron://dendron.dendron-site/dendron.topic.search]]
```

Comment:
the note reference doesn't make sense. 
- "lookup and Finding Notes", remove "Finding Notes" since its a link to "Lookup"
- the command is a mouthful, would say "this can be done by running "Search: Find in Files (Shortcut...)"...

```
Most queries can be satisfied using lookup and Finding Notes, but sometimes you want to do a full text search across the contents of your notes. This can be done with Ctrl+Shift+F / Cmd+Shift+F, or Search: Find in Files from the command palette, which leverages VSCode built-in search capabilities.
```